lib/gui/pages/main/templates/main.tpl.html

```diff
@@ -95,8 +95,8 @@
         </progress-button>
 
         <p class="step-footer step-footer-split" ng-show="main.state.getFlashState().speed && main.state.getFlashState().percentage != 100">
-          <span>ETA: {{ main.state.getFlashState().eta | secondsToDate | amDateFormat:'m[m]ss[s]' }}</span>
           <span ng-bind="main.state.getFlashState().speed.toFixed(2) + ' MB/s'"></span>
+          <span>ETA: {{ main.state.getFlashState().eta | secondsToDate | amDateFormat:'m[m]ss[s]' }}</span>
```

Comment:
Off-Topic: relying on date formatting to display a "time left ETA" looks a bit ugly, but I guess as long as it works... ;-)

Hopefully this doesn't mean that the ETA will go haywire if somebody happens to be flashing an image with Etcher when the clocks go forwards or backwards? (DST changes)